egs/cnceleb/v1/run.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+# Copyright   2017   Johns Hopkins University (Author: Daniel Garcia-Romero)
+#             2017   Johns Hopkins University (Author: Daniel Povey)
+#        2017-2018   David Snyder
+#             2018   Ewald Enzinger
+#             2019   Tsinghua University (Author: Jiawen Kang and Lantian Li)
+# Apache 2.0.
+#
+# This is an i-vector-based recipe for CN-Celeb database.
+# See ../README.txt for more info on data required. The recipe uses
+# CN-Celeb/dev for training the UBM, T matrix and PLDA, and CN-Celeb/eval
+# for evaluation. The results are reported in terms of EER and minDCF,
+# and are inline in the comments below.
+
+. ./cmd.sh
+. ./path.sh
+set -e
+mfccdir=`pwd`/_mfcc
+vaddir=`pwd`/_vad
```

Comment:
I don't think I've seen these directories preceded by an underscore before... It seems unusual in Kaldi recipes. I would consider changing that to be consistent with the other recipes.

Reply:
Thanks a lot. I have changed that to be consistent with previous recipes.